src/app/CommandHandler.h

```diff
@@ -416,7 +426,8 @@ class CommandHandler : public Messaging::ExchangeDelegate
 
     bool mSentStatusResponse = false;
 
-    State mState = State::Idle;
+    State mState       = State::Idle;
+    bool mGroupRequest = false;
```

Comment:
This makes CommandHandler bigger, due to padding, when we didn't have to do that (could have added it _after_ the TLV writer bits, next to the other two booleans).

This also does not handle commands that try to send a data response correctly, right?  Is there a plan for an actual correct fix for this?

Reply:
> This makes CommandHandler bigger, due to padding, when we didn't have to do that (could have added it after the TLV writer bits, next to the other two booleans).

Right, I haven't noticed it. 

> This also does not handle commands that try to send a data response correctly, right? Is there a plan for an actual correct fix for this?

Why? I don't think I understand this part. The flag is set `true` for all group messages that do not require sending response and it is set to `false` for non-group messages that should send a data response. What should be the actual correct fix?

Reply:
The flag is _set_ fine, but it's only _checked_ in AddStatus.  There are other methods that add data responses where it's not checked....

Reply:
As we discussed offline, VerifyOrDie check in `AddStatus` was the only reason of observed failures. The responses prepared for other types of messages like data reponses will be dropped from the queue anyway.